Realm/Realm/DatabaseTypes/Accessors/ManagedAccessor.cs

```diff
@@ -199,16 +221,31 @@ private void RaisePropertyChanged(string propertyName = null)
             _onNotifyPropertyChanged(propertyName);
         }
 
+        /// <summary>
+        /// Returns all the objects that link to this object in the specified relationship.
+        /// </summary>
+        /// <param name="objectType">The type of the object that is on the other end of the relationship.</param>
+        /// <param name="property">The property that is on the other end of the relationship.</param>
+        /// <returns>A queryable collection containing all objects of <c>objectType</c> that link to the current object via <c>property</c>.</returns>
+        [Obsolete("Use realmObject.DynamicApi.GetBacklinksFromType() instead.")]
         public IQueryable<dynamic> GetBacklinks(string objectType, string property) => DynamicApi.GetBacklinksFromType(objectType, property);
```

Comment:
I'm not sure how I feel about this - we're obsoleting functionality that nobody should be using, can't we just delete it?

Reply:
We definitely can, I was trying to be too conservative probably

Reply:
If we delete it then it'll be breaking change. because this is used by the weaved classes too.